test/crawlers/cheerio_crawler.test.js

```diff
@@ -605,18 +605,27 @@ describe('CheerioCrawler', () => {
             expect(proxies[3]).toEqual(shuffled[0]);
         });
 
-        test('should work with useApifyProxy', async () => {
+        test('should work with proxyConfiguration', async () => {
             process.env[ENV_VARS.PROXY_PASSWORD] = 'abc123';
             const proxies = [];
-            const useApifyProxy = true;
-
-            const proxy = Apify.getApifyProxyUrl();
+            const utilsRequestMock = sinon.mock(utilsRequest);
+            const status = JSON.stringify({ connected: 'true' });
+            const proxyUrl = 'http://auto:abc123@proxy.apify.com:8000';
+            const url = 'http://proxy.apify.com/?format=json';
+            const countryCode = undefined;
```

Comment:
Does it need this? Shouldn't omitting the property be enough?

Reply:
Yes, it should. Simplified! :)